src/components/display/fewlines/Button.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from "styled-components";
+
+import { ButtonVariant } from "../../../@types/ButtonVariant";
+
+interface ButtonProps {
+  variant?: ButtonVariant;
+}
+
+export const Button = styled.button<ButtonProps>`
+  height: 4rem;
+  border-radius: 0.2rem;
```

Comment:
Use the first value of the theme's `radii` list.

Reply:
OK fixed 7e41855